File: app-presentation/src/main/java/com/teewhydope/app/presentation/login/model/LoginResponsePresentationModel.kt
package com.teewhydope.app.presentation.login.model

data class LoginResponsePresentationModel(
    val userData: UserDataPresentationModel,
    val smartSaverTransactions: List<SmartSaverTransactionPresentationModel>
)

data class SmartSaverTransactionPresentationModel(
    val amount: Long,
    val type: String,
    val narration: String,
    val dateCreated: String
)

data class UserDataPresentationModel(
    val firstName: String,
    val smartSaverBalance: Long,
    val greenSaverBalance: Long,
    val fixedDepositBalance: Long,
)

File: iosApp/iosApp/Presentation/Login/Model/LoginResponsePresentationModel.swift
//
//  LoginResponsePresentationModel.swift
//  iosApp
//
//  Created by Animasahun Ibrahim on 17/03/2024.
//  Copyright © 2024 orgName. All rights reserved.
//

import Foundation

struct LoginResponsePresentationModel{
    let userData: UserDataPresentationModel
    let smartSaverTransactions: [SmartSaverTransactionPresentationModel]
}

struct SmartSaverTransactionPresentationModel{
    let amount: Int64
    let type: String
    let narration: String
    let dateCreated: String
}

struct UserDataPresentationModel {
    let firstName: String
    let smartSaverBalance: Int64
    let greenSaverBalance: Int64
    let fixedDepositBalance: Int64
}

